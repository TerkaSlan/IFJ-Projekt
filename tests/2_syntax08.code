/*Should be synax error becasue of undeclared return type of run*/

class Main {

	static run() {
		
	}
}