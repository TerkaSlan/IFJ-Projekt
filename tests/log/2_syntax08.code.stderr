ERROR: Syntactic - Line: 5 - Type declaration expected.
